Firestore/Swift/Source/Result/CollectionReference+Result.swift

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2019 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import FirebaseFirestore
+
+#if swift(>=5.0)
+
+  extension CollectionReference {
+    /// Adds a new document to this collection with the specified data, assigning it a document ID automatically.
+    ///
+    /// - Parameters:
+    ///   - data: The `Dictionary` containing the data for the new document.
+    ///   - completion: The closure to execute on successfully writing to the server or on receipt of an error.
+    ///   It will not be called while the client is offline, though local changes will be visible immediately.
+    ///   - result: The result of request. On success it is empty, otherwise it contains an `Error`.
```

Comment:
I just did in this way because without the result description there is an ugly no-description label in autogenerated docs
<img width="254" alt="Schermata 2020-11-12 alle 15 54 09" src="https://user-images.githubusercontent.com/40797951/98956314-27a0a800-2500-11eb-970e-1280bc18c4d6.png">
<img width="493" alt="Schermata 2020-11-12 alle 15 59 39" src="https://user-images.githubusercontent.com/40797951/98956317-2a030200-2500-11eb-8f55-6f55b4f8ce78.png">
<img width="517" alt="Schermata 2020-11-12 alle 16 03 02" src="https://user-images.githubusercontent.com/40797951/98956626-8d8d2f80-2500-11eb-93d1-2026b652dc43.png">
Any idea to solve this?